src/common/transformations/src/transformations/common_optimizations/push_constant_to_subgraph.cpp

```diff
@@ -0,0 +1,95 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/common_optimizations/push_constant_to_subgraph.hpp"
+
+#include <openvino/core/validation_util.hpp>
+#include <openvino/op/util/multi_subgraph_base.hpp>
+
+#include "itt.hpp"
+
+bool ov::pass::PushConstantToSubgraph::run_on_model(const std::shared_ptr<Model>& model) {
```

Comment:
I would recommend to split this big method to small functions, e.g.
try_const_fold, update_description, update_inputs and so on

Reply:
done